sstables/sstable_set.cc

```diff
@@ -24,6 +24,7 @@
 #include "compatible_ring_position.hh"
 #include "compaction_strategy_impl.hh"
 #include "leveled_compaction_strategy.hh"
```

Comment:
Please add a nice comment to the changelog. If you don't have inspiration, you can run a  LIMIT 1 query on TWCS with and without and compare reactor_aio_bytes_read changes before and after. The ratio should be O(number of windows).